notifier/notifier.go

```diff
@@ -397,6 +398,25 @@ func (n *Notifier) Alertmanagers() []*url.URL {
 	return res
 }
 
+// DroppedAlertmanagers returns a slice of Alertmanager URLs.
+func (n *Notifier) DroppedAlertmanagers() []*url.URL {
+	n.mtx.RLock()
+	amSets := n.alertmanagers
```

Comment:
s/alertmanagers/droppedAlertmanagers/?

Reply:
Missed it when resolving merge conflicts, thanks!